stbt-run

```diff
@@ -22,7 +25,7 @@ parser.add_argument(
     '--save-video', help='Record video to the specified file', metavar='FILE',
     default=stbt.get_config('run', 'save_video'))
 parser.add_argument(
-    '--save-trace', metavar='FILE', default=None,
+    '--save-trace', metavar='FILE', nargs='?', const=TO_SOCKET,
```

Comment:
I agree that having to have a sentinel isn't great, but if the tracing is conditional on if `STBT_TRACING_SOCKET` is set then
`STBT_TRACING_SOCKET=x stbt run --save-trace -- test.py` and
`STBT_TRACING_SOCKET=x stbt run test.py`
will both result in the tracing being activated, which is then particularly troublesome if we or someone else were to push our own tracing functionality upstream, resulting in:
`STBT_TRACING_SOCKET=x stbt run --custom-trace=trace.log test.py`
where some secret, internal decision has to be be made about which trace takes priority.

Perhaps a better solution is to have separate, mutually exclusive command line arguments, one which writes to a file and one which writes to a socket.